plugins/generator-1.14.4/forge-1.14.4/procedures/world_use_bonemeal.java.ftl

```diff
@@ -0,0 +1,5 @@
+{
+	if (BoneMealItem.applyBonemeal(new ItemStack(Items.BONE_MEAL), (World) world, new BlockPos((int) ${input$x}, (int) ${input$y}, (int) ${input$z})) ||
+	BoneMealItem.growSeagrass(new ItemStack(Items.BONE_MEAL), (World) world, new BlockPos((int) ${input$x}, (int) ${input$y}, (int) ${input$z}), (Direction) null)) {
+	}
```

Comment:
You can add the bonemeal particles by adding 
```java
if (!((World) world).isRemote)
	((World) world).playEvent(2005, new BlockPos((int) ${input$x}, (int) ${input$y}, (int) ${input$z}), 0);
```
inside the if statement. Otherwise, you can remove it and call `applyBonemeal` and `growSeagrass` as if they were void methods.